packages/node-resizin/README.md

```diff
@@ -26,7 +26,7 @@ yarn add node-resizin
 ## Quick start
 
 ```js
-const { uploadFactory } = require('node-resizin');
+const uploadFactory = require('node-resizin');
```

Comment:
this import is changed from named import to default import - but I cant see any change in the code or update version in package.json

It was an old mistake from before just here in the documentation or what?

Reply:
It was a bug in documentation caused probably by Copy/Paste. If you're importing uploadFactory from `resizin` package it's named import, but from `node-resizin` it's default import as there is nothing else in node package.